cylc/flow/task_job_mgr.py

```diff
@@ -1087,8 +1090,7 @@ def _prep_submit_task_job(
         Returns:
             * itask - preparation complete.
             * None - preparation in progress.
-            * False - perparation failed.
-
+            * False - preparation failed.
```

Comment:
(there should be a blankline after a top-level section in a docstring):

```suggestion
            * False - preparation failed.

```